src/sentry/feedback/usecases/create_feedback.py

```diff
@@ -0,0 +1,48 @@
+import datetime
+from uuid import uuid4
+
+from sentry.issues.grouptype import FeedbackGroup
+from sentry.issues.issue_occurrence import IssueEvidence, IssueOccurrence
+from sentry.issues.producer import produce_occurrence_to_kafka
+from sentry.utils.dates import ensure_aware
+
+
+def make_evidence(feedback):
+    evidence_data = {}
+    evidence_display = []
+    if feedback.get("contact_email"):
+        evidence_data["contact_email"] = feedback["contact_email"]
+        evidence_display.append(
+            IssueEvidence(name="contact_email", value=feedback["contact_email"], important=True)
+        )
+    if feedback.get("message"):
+        evidence_data["message"] = feedback["message"]
+        evidence_display.append(
+            IssueEvidence(name="message", value=feedback["message"], important=True)
+        )
+    return evidence_data, evidence_display
+
+
+def create_feedback_issue(event, project_id):
+    event["event_id"] = event.get("event_id") or uuid4().hex
+    evidcence_data, evidence_display = make_evidence(event["feedback"])
+    occurrence = IssueOccurrence(
+        id=uuid4().hex,
+        event_id=event.get("event_id") or uuid4().hex,
+        project_id=project_id,
+        fingerprint=[
+            uuid4().hex
+        ],  # random UUID for fingerprint so feedbacks are grouped individually
+        issue_title="User Feedback",
```

Comment:
How will this be displayed?  Will a user see "User Feedback" repeated over and over again?  Maybe a better system would be to have the message here and the subtitle be the location of the feedback.  I'm not sure "User Feedback" is a meaningful title if the page is dedicated to user feedback.

Reply:
This is a good point. I'll add a comment here. The intention is for these "Issues" to be hidden in the issues stream UI eventually, while our Feedback UI will display all the information correctly, the "Title" isn't super important atm.

Reply:
comment added!